ocs_ci/ocs/resources/pod.py

```diff
@@ -819,6 +819,48 @@ def get_ocs_operator_pod(ocs_label=constants.OCS_OPERATOR_LABEL, namespace=None)
     return ocs_operator_pod
 
 
+def get_noobaa_operator_pod(
+    ocs_label=constants.NOOBAA_OPERATOR_POD_LABEL, namespace=None
```

Comment:
we can remove the ocs_label from the function argument as it is a constant and can be directly given on line 838

Reply:
Yeah it can be added, with this code it reduces the line length.